packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -92,6 +92,10 @@ export interface ParticipantItemProps {
    * Optional strings to override in component
    */
   strings?: Partial<ParticipantItemStrings>;
+  /**
+   * Optional callback when component is clicked. If this is defined then contextual menu will not shown when component is clicked.
```

Comment:
nit typo shown->show:
```suggestion
   * Optional callback when component is clicked. If this is defined then contextual menu will not show when component is clicked.
```

Reply:
This comment says that contextual menu will not be shown when this is defined.

But I don't see how that's done in the logic below. `menuHidden` changes the same way as before...

Reply:
Sorry, my mistake. I was planning on not showing the contextual menu when `onClick` is defined but decided not to in the end  because I think would be "anti-Fluent UI" behavior. Contextual menu is only shown when there are menuItems. Will fix this comment.